source/MaterialXView/ShaderValidators/Glsl/GlslValidator.cpp

```diff
@@ -581,21 +784,115 @@ void GlslValidator::bindTimeAndFrame()
     }
 }
 
-
-void GlslValidator::bindViewInformation()
+void GlslValidator::bindLighting()
 {
     ShaderValidationErrorList errors;
 
     if (_programId <= 0)
     {
-        const std::string errorType("GLSL input binding error.");
+        const std::string errorType("GLSL light binding error.");
         errors.push_back("Cannot bind without a valid program");
         throw ExceptionShaderValidationError(errorType, errors);
     }
 
-    // Pull information from HwShader
-    if (_hwShader)
+    // Bind a couple of lights if can find the light information
```

Comment:
Binding works for dir + point. Need to fix up locations of point to get non-black.